CHANGES.txt

```diff
@@ -5,6 +5,8 @@ Changes for crash
 Unreleased
 ==========
 
+ - Add support for pasting of multiple statements
```

Comment:
suggestion: Added support for pasting and executing multiple statements at once.